src/repository/manifest.rs

```diff
@@ -86,7 +87,10 @@ impl Manifest {
         // |   3. In the rpkiManifest, thisUpdate precedes nextUpdate.
 
         if self.content.this_update >= self.content.next_update {
-            return Err(ValidationError)
+            // XXX This should be checked during decoding.
+            return Err(
+                VerificationError::new("thisUpdate after nextUpdate").into()
```

Comment:
The test says >=, not >, so is this error message strictly correct?

Reply:
I would suggest to say something like:
```
format!(
  "Manifest thisUpdate {} is not before nextUpdate {}",
  self.content.this_update.to_rfc3339(),
  self.content.next_update.to_rfc3339()
)
```

As this is only called when needed, and provides a bit more context.

Reply:
Turns out this is already checked during decoding and the whole thing can go.